.github/workflows/agw-workflow.yml

```diff
@@ -213,6 +213,17 @@ jobs:
         run: |
           echo "Available storage:"
           df -h
+      - name: Notify failure to slack
```

Comment:
Should a notification also be added to `Run coverage with Bazel (COMMON / SESSIOND / SCTPD / LIAGENT / CONNECTIOND)`?

Reply:
We can add the functionality to that job too, but it already has a "Notify failure to slack" job (that is restricted to failure in step `c-cpp-codecov-upload`). What would be the best?:
1) Just name it something different (e.g.: "Notify failure to slack (#bazel-ci)"
2) Rename the other job too

Reply:
discussed offline